packages/design-system-tailwind-preset/src/typography.ts

```diff
@@ -105,29 +72,18 @@ export const typography = {
     's-heading-lg': 'var(--typography-s-heading-lg-font-weight)',
     's-heading-md': 'var(--typography-s-heading-md-font-weight)',
     's-heading-sm': 'var(--typography-s-heading-sm-font-weight)',
-    's-body-lg-medium': 'var(--typography-s-body-lg-medium-font-weight)',
-    's-body-lg-regular': 'var(--typography-s-body-lg-regular-font-weight)',
-    's-body-md-bold': 'var(--typography-s-body-md-bold-font-weight)',
-    's-body-md-medium': 'var(--typography-s-body-md-medium-font-weight)',
+    's-body-lg': 'var(--font-weight-regular)',
```

Comment:
`--typography-l-body-lg-regular-letter-spacing` CSS token incorrectly points to medium? so falling back to root font weight token here too

https://github.com/MetaMask/design-tokens/blob/main/src/css/typography.css#L73